ios/RNFishjamClientModule.swift

```diff
@@ -69,136 +69,137 @@ public class MembraneWebRTCModule: Module {
             "SendMediaEvent",
             "BandwidthEstimation")
 
-        let membraneWebRTC: MembraneWebRTC = MembraneWebRTC { (eventName: String, data: [String: Any]) in
+        let rnFishjamClient: RNFishjamClient = RNFishjamClient { (eventName: String, data: [String: Any]) in
             self.sendEvent(eventName, data)
         }
 
-        AsyncFunction("create") {
-            try membraneWebRTC.create()
+        AsyncFunction("connect") { (url: String, peerToken: String, promise: Promise ) in
```

Comment:
why connect does not now take endpont metadata as an argument?

Reply:
old membrane webrtc was really a RTC Engine client so:
- the user took care of connecting to the websocket (or something else - it's up to the user)
- connect function took endpoint metadata and it was joining to the room
Now that we have only fishjam client user no longer needs to care about websockets - we do it for them
So:
- the connect function connects to the websocket and authenticates so it takes url and peerToken
- the joinRoom function is like the old connect function and joins the room with peer metadata